src/main/java/dto/CustomerDetails.java

```diff
@@ -0,0 +1,40 @@
+package dto;
+
+import java.math.BigDecimal;
+
+import javax.validation.Valid;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+public class CustomerDetails {
+	@Valid
+	@NotNull(message = "3")
+	private String company;
+	@Valid
```

Comment:
Also

Reply:
Yes and you twice wrote "@NotNull(message = "3")"